Cargo.toml

```diff
@@ -16,7 +16,7 @@ repository = "https://github.com/hannobraun/3mf-rs"
 
 
 [dependencies]
-quick-xml = { version = "0.27.1", features = ["serialize"] }
+quick-xml = { version = "0.28.1", features = ["serialize"] }
```

Comment:
Why `0.28.1` specifically, when `0.28.2` [is the latest version](https://crates.io/crates/quick-xml)?

Reply:
Weird, for some reason VS Code told me the last one is `0.28.1` D:
![image](https://user-images.githubusercontent.com/13365789/233646030-fe354162-fffc-4b08-a913-4457f0cb8b5c.png)

Reply:
Oh well :smile: